app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -228,28 +228,6 @@ class AttendeeViewModel(private val attendeeService: AttendeeService, private va
             message.value = "Unable to create Order!"
         }
     }
-    
-    fun createOrder(order: Order) {
-        compositeDisposable.add(orderService.placeOrder(order)
```

Comment:
why is this method removed

Reply:
Because there is another method just above it with the same name and this method is not being used anywhere.